src/lib/components/ApplicationDetail/index.svelte

```diff
@@ -1,13 +1,43 @@
 <script lang="ts">
+	import { providerStore } from 'web3-stores';
+	import { workstreamStore } from '$lib/stores/workstreamsStore.js';
+
 	import Modal from '$lib/shared/Modal.svelte';
 	import User from '$lib/shared/User.svelte';
+
 	export let application;
+	let workstream;
+
+	function getWorkstream() {
+		return (workstream = $workstreamStore.find(
+			(workstream) => workstream.id === application.workstream_id
+		));
+	}
+	getWorkstream();
```

Comment:
If you want to call `getWorkstream` once the site as loaded, I recommend to do this inside `onMount`.